src/PasswordRules.php

```diff
@@ -36,6 +36,23 @@ public static function changePassword($username, $oldPassword = null)
         return $rules;
     }
 
+    public static function optionallyChangePassword($username, $oldPassword = null)
+    {
+        $rules = self::changePassword($username, $oldPassword);
+
+        $rules = array_merge($rules, [
+            'nullable',
+        ]);
+
+        foreach ($rules as $key => $rule) {
+            if (is_string($rule) && $rule === 'required') {
```

Comment:
What are the changes that someone would use `REQUIRED` rather than `required`?

```is_string($rule) && strtolower($rule) === 'required'```

Reply:
Not sure, I've always used lowercase for Laravel string-based validation rules. Does Laravel support alternative cases for these?

Reply:
Looks like Laravel convert string based rules to studly case. https://github.com/laravel/framework/blob/5.7/src/Illuminate/Validation/ValidationRuleParser.php#L239

However, this isn't a problem in this case as the `$rules` array referenced here is not supplied by the user of the package.